services/app-api/utils/kafka/kafka-source-lib.ts

```diff
@@ -17,6 +17,7 @@ type KafkaPayload = {
 type SourceTopicMapping = {
   sourceName: string;
   topicName: string;
+  s3Prefix?: string;
 };
```

Comment:
@BearHanded like so?

Reply:
The chunk down around line 113, determineS3TopicName will need to make use of that information as well. It will need to determine the topic name based on both teh source & prefix, if a prefix exists.